### ISSUE TITLE
Support multisites in `install.sh`

### ISSUE THREAD

**Issue by BWPanda** [backdrop/backdrop-issues#4151]
**Description of the need**
If you want to install Backdrop from the command line using the core `install.sh` script, you can only do a regular install. You can't do a multisite install.

Setting up a Backdrop installation for multisite, and then accessing the GUI installer works fine - the `/sites/[SITE]/settings.php` is updated with the DB info, config directories, etc.
But setting up the same Backdrop multisite installation, and then accessing the command line installer, doesn't work - the core `/settings.php` is updated with the DB info, config directories, etc., not the site-specific one.

**Proposed solution**
The command line installer should work the same as the GUI one; however it is that the GUI one knows to check for site-specific directories, the command line one should do the same.

**Alternatives that have been considered**
For now multisites have to be setup via the GUI, but that's not always possible (e.g. setting up a non-interactive test environment).

**Additional information**
To test, here's a Lando file for setting up the multisite environment:

```yaml
name: multisite
recipe: backdrop
config:
  webroot: www
services:
  appserver:
    build:
      - git clone https://github.com/backdrop/backdrop.git /app/www
      - mkdir /app/www/sites/multisite && cp /app/www/settings.php /app/www/sites/multisite/
```
I couldn't do this as part of the Lando setup, so one you run `lando start` you have to manually add this to the `/sites/sites.php` file:
`$sites['multisite.lndo.site'] = 'multisite';`

---

PR by @BWPanda: https://github.com/backdrop/backdrop/pull/2946

**Comment by BWPanda** [comment]
Here's a PR that fixes this for me: https://github.com/backdrop/backdrop/pull/2946

It adds a `url` option to the install script so you can specify the URL of the site you're installing (as defined in `sites.php`), and then makes a change in `find_conf_path()` that removes the leading `.` from the script path (present when running the install script: `./core/scripts/install.sh`) which was causing issues with finding the right site directory.

**Comment by indigoxela** [comment]
Could the following work in a Lando file?

`echo '$sites["multisite.lndo.site"] = "multisite";' > /app/www/sites/sites.php`

**Comment by BWPanda** [comment]
No, I tried that but the redirection doesn't work: https://github.com/lando/lando/issues/1832

**Comment by quicksketch** [comment]
I posted some feedback on the PR: https://github.com/backdrop/backdrop/pull/2946#pullrequestreview-310190326

Passing the URL in definitely seems like it could work. Perhaps we could also potentially support passing in the site directory name rather than the URL? I'm not sure if that would be any better or easier.

**Comment by BWPanda** [comment]
@quicksketch Thanks for the review. I updated the PR with a change as requested, and answered your other question too.

As for passing in the site directory instead of the URL, every other place I've seen multisite support (which is pretty much just Drush and other command-line tools) they use the URL, not the directory name. So I just copied that because that's likely what people are used to if they've used Drush, etc. before.
